api/SVGAElement.json

```diff
@@ -204,43 +204,43 @@
         "__compat": {
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": "38"
```

Comment:
I'm unable to confirm this. What's your information source?

Reply:
I was kinda guessing, by looking at this: https://www.chromestatus.com/features/5743723954569216 and assuming SVG attrs might be the same.

Reply:
I figured it out. You've got the Opera number here instead of the Chrome number.